docs/developer.rst

```diff
@@ -52,13 +51,20 @@ To build ``jaxlib`` without CUDA GPU support (CPU only), drop the ``--enable_cud
 
   python build/build.py
   pip install -e build  # installs jaxlib (includes XLA)
-  pip install -e .      # installs jax
 
+If you're only modifying Python portions of JAX, you may be able to skip this
```

Comment:
I would probably try to move this text higher up, before the instructions on how to build `jaxlib`.

Reply:
Done.